src/main/java/org/fogbowcloud/arrebol/execution/dockerworker/ContainerRequestHelper.java

```diff
@@ -0,0 +1,93 @@
+package org.fogbowcloud.arrebol.execution.dockerworker;
+
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.log4j.Logger;
+import org.fogbowcloud.arrebol.utils.AppUtil;
+import org.json.JSONObject;
+import org.fogbowcloud.arrebol.execution.constans.DockerConstants;
+
+import java.io.UnsupportedEncodingException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ContainerRequestHelper {
+    private HttpWrapper httpWrapper;
+    private String address;
+    private String image;
+    private String containerName;
+    private Map<String, String> requirements;
+
+    private final Logger LOGGER = Logger.getLogger(ContainerRequestHelper.class);
+
+    public ContainerRequestHelper(String address, String containerName, String image) {
+        this.httpWrapper = new HttpWrapper();
+        this.address = address;
+        this.containerName = containerName;
+        this.image = image;
+        this.requirements = new HashMap<>();
+    }
+
+    public String createContainer() throws Exception {
+        final String endPoint = String.format("%s/containers/create?name=%s", address, containerName);
+        StringEntity body = jsonCreateContainer();
+        String response = this.httpWrapper.doRequest(HttpPost.METHOD_NAME, endPoint, body);
+        String containerId = AppUtil.getValueFromJsonStr("Id", response);
```

Comment:
I'm afraid that we are not dealing with errors here in the REST calls. Maybe, add issues in the github and FIXMEs or TODOs in code